python/lsst/pipe/analysis/plotUtils.py

```diff
@@ -4,14 +4,12 @@
 import numpy as np
 
 import lsst.afw.cameraGeom as cameraGeom
-import lsst.afw.geom as afwGeom
-import lsst.geom as lsstGeom
 import lsst.afw.image as afwImage
 import lsst.afw.table as afwTable
+import lsst.daf.persistence as dafPersist
```

Comment:
Is this one necessary?

Reply:
Arghhh...thanks for catching that!  Snuck in on a conflicted cherry pick...